CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ that can be found in the LICENSE file. -->
 
 To know more about breaking changes, see the [Migration Guide][].
 
+## 3.0.0-dev.5
+
+- Fix: the requestPermissionExtend method returns the wrong result on android API 34.
```

Comment:
```suggestion
### Fixes

- Fix `requestPermissionExtend` returns the incorrect status on Android API 34.
```

Reply:
The same tag will affect the generation of markdown toc and trigger markdown lints. I personally do not recommend using multiple identical secondary headers.

Reply:
Although, I turned off this lint, the repeated headers have a negative effect on toc generation.

https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md024.md

Reply:
Well, we don't need a TOC for the CHANGELOG. :D